bbc_sale/views/product.xml

```diff
@@ -146,5 +146,17 @@
             </field>
         </record>
 
+        <record id="website_sale_product_template_form_view_inherited_is_synced_magento" model="ir.ui.view">
```

Comment:
Can this be added to the existing product_template_form_view instead of adding a new view definition? The existing view already has a very 'high' priority so it is bound to already include the override from website_sale.